aws/policy/data-services.yaml

```diff
@@ -110,3 +110,41 @@ Statement:
       ForAnyValue:StringEquals:
         iam:AWSServiceName:
           - 'elasticache.amazonaws.com'
+  - Sid: KafkaConfiguration
```

Comment:
Done, please check.

Reply:
My apologies, I introduced a typo in my suggestion - it should be `arn:aws:iam::{{ aws_account_id }}:role/aws-service-role/kafka.amazonaws.com/AWSServiceRoleForKafka`  (only 1 : after the aws_account_id template). That's my fault!
This looks good with that updated.

I normally only work through this PR queue once a week since everything has to be manually tested and deployed but if you ping me in `#ansible-aws` on freenode I should have availability in the next few days to re-review this one faster.

Reply:
Fixed.